charts/core-dump-handler/values.openshift.yaml

```diff
@@ -0,0 +1,12 @@
+# RHCOS requires a different host path, /mnt is writable
+daemonset:
+  hostDirectory: "/mnt/core-dump-handler"
+  coreDirectory: "/mnt/core-dump-handler/cores"  
+  vendor: default # default is a rhel8 build, alternatively set rhel7
+
+serviceAccount:
+  create: true
+  name: "core-dump-admin"
+  # when running on Openshift, this creates a new SCC and attaches it to the service account
+  createScc: true
+  sccName: "core-dump-admin-privileged"
```

Comment:
I like the idea more now I see it. :+1:

Reply:
I will probably create a specific file for ROKS so the vendor value is set and it's a consistent set of instructions.

Reply:
very good, I just also add the `composerCrioImageCmd` into this one - which I forgot